.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -19,3 +19,15 @@ One of the following will happen to feature requests:
     see the [contributor guide](https://github.com/seqan/seqan3/pull/CONTRIBUTING.md).
   * The feature request is invalid and will be closed.
 -->
+
+### Done
+
+- [ ] Implementation and design approved
+- [ ] Unit tests pass
+- [ ] Test coverage > 97%
+- [ ] Micro benchmarks implemented
+- [ ] API documentation added
+- [ ] Tutorial/teaching material added
+- [ ] Macro benchmark with <= 10 % performance difference to SeqAn2 (if applicable)
+- [ ] Tests compile in less than 30 seconds
```

Comment:
On what machine? :) Also the Debug tests or only in Release?

Reply:
Well, ok we can specify a mode Debug/Release. The question would be what does @seqan/core think about 30 seconds. Is it too much or too low?

Reply:
> On what machine? :) 

Your desktop/laptop computer for a single test case. I guess the differences between the CPUs aren't that big between the developers in the team. But if required we might find a minimum CPU spec and use that. But these things should not become to complicated, otherwise they will be useless if the team starts to ignore them.